engine/app/controllers/good_job/dashboards_controller.rb

```diff
@@ -1,8 +1,56 @@
 module GoodJob
   class DashboardsController < GoodJob::BaseController
+    class JobFilter
+      attr_accessor :params
+      def initialize(params)
+        @params = params
+      end
+
+      def last
+        @last ||= jobs.last
+      end
+
+      def jobs
+        sql = GoodJob::Job.display_all(after_scheduled_at: params[:after_scheduled_at], after_id: params[:after_id])
+          .limit(params.fetch(:limit, 10))
+        if params[:job_class]
+          sql = sql.where("serialized_params->>'job_class' = ?", params[:job_class])
+        end
+        if params[:state]
+          case params[:state]
+          when 'finished'
+            sql = sql.finished
+          when 'unfinished'
+            sql = sql.unfinished
+          when 'errors'
+            sql = sql.where.not(error: nil).count
```

Comment:
I believe the `count` should not be added

Reply:
oh, copy&paste error from the facets. fixed it.